torchci/lib/clickhouse.ts

```diff
@@ -1,5 +1,12 @@
+// This file can't be imported by .tsx files due to client/server stuff (ex not
```

Comment:
This can only be used on the server side, i.e. API.  All HUD pages should query the API https://github.com/pytorch/test-infra/tree/main/torchci/pages/api/clickhouse to get the data instead.  This is the same reason I need to add the toggle for CH/Rockset on HUD metric page

Reply:
Maybe something that can be done by a linter?